src/librustc_hir/def.rs

```diff
@@ -451,13 +451,19 @@ impl<Id> Res<Id> {
         }
     }
 
-    pub fn matches_ns(&self, ns: Namespace) -> bool {
+    /// Returns `None` if this is `Res::Err`
```

Comment:
```suggestion
    /// Returns `None` if this is `Result::Err`
```

I believe `Res` means `Result`?